src/js/sign.js

```diff
@@ -0,0 +1,221 @@
+        wrongLengthPwSpan.classList.add('pwErrorSpan');
+        msgCreate(
+          wrongLengthPwSpan,
+          pwDiv,
+          input,
+          '비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.'
+        );
+        //8글자 이상, 비밀번호 형식 안맞을 경우
+      } else {
+        if (!checkSpecial(newMsg) || !checkEng(newMsg) || !checkNum(newMsg)) {
+          if (pwDiv.children.length !== 2) pwDiv.lastElementChild.remove();
+          const wrongLengthPwSpan = document.createElement('span');
+          wrongLengthPwSpan.classList.add('pwErrorSpan');
+          msgCreate(
+            wrongLengthPwSpan,
+            pwDiv,
+            input,
+            '비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.'
+          );
+        }
+        //8글자 이상, 비밀번호칸 두개 서로 일치하지 않을 경우
+        if (
+          pwSignupInput2.value &&
+          pwSignupInput1.value !== pwSignupInput2.value
+        ) {
+          if (pwSignupDiv2.children.length !== 2)
+            pwSignupDiv2.lastElementChild.remove();
+          const wrongLengthPwSpan = document.createElement('span');
+          wrongLengthPwSpan.classList.add('pwDisMatchSpan');
+          msgCreate(
+            wrongLengthPwSpan,
+            pwSignupDiv2,
+            pwSignupInput2,
+            '비밀번호가 일치하지 않습니다.'
+          );
+        }
+      }
+    }
+  }
+}
+
+//비밀번호 표시
+function viewPassword(e) {
+  const input = e.target.parentElement.firstElementChild;
+  if (input.type == 'password') {
+    input.setAttribute('type', 'text');
+    e.target.setAttribute('src', './img/eye-on.svg');
+  } else {
+    input.setAttribute('type', 'password');
+    e.target.setAttribute('src', './img/eye-off.svg');
+  }
+}
+
+//로그인 / 회원가입 버튼 눌렀을 때 이메일 비밀번호 일치 확인
+function loginCheck(e) {
+  //이메일 비밀번호 일치할때
+  if (
+    !pwSignupDiv1 &&
+    emailInput.value === VERIFYEAMIL &&
+    pwInput.value === VERIFYPW
+  ) {
+    location.replace('./folder.html');
+  } else if (
+    pwSignupDiv1 &&
+    emailDiv.children.length == 2 &&
+    pwSignupDiv1.children.length == 2 &&
+    pwSignupDiv2.children.length == 2 &&
+    pwSignupInput1.value &&
+    pwSignupInput2.value &&
+    emailInput.value
+  ) {
+    // console.log('오ㅓ류없었냐ㅑㅑㅑㅑㅑㅑㅑㅑ');
+    location.replace('./folder.html');
+  } else if (pwSignupDiv1) {
+    //이메일 비밀번호 불일치일때
+    // console.log('z번');
+    //아래는 오류가 있거나 비어있는 칸이 있으면 확인해 달라 오류 뜨게하는 코드
+    //
+    //이메일칸에 내용이 없거나 오류메세지가 있는 경우
+    if (!emailInput.value || emailDiv.children.length !== 2) {
+      msgDelete('.errorEmailMsg', '.wrongEmailSpan');
+      const noEmailspan = document.createElement('span');
+      noEmailspan.classList.add('wrongEmailSpan', 'wrongMsg');
+      msgCreate(noEmailspan, emailDiv, emailInput, '이메일을 확인해 주세요.');
+    }
+    //비밀번호칸에 내용이 없거나 오류메세지가 있는 경우
+    if (!pwSignupInput1.value || pwSignupDiv1.children.length !== 2) {
+      if (pwSignupDiv1.children.length !== 2)
+        pwSignupDiv1.lastElementChild.remove();
+      const noPwSignupspan1 = document.createElement('span');
```

Comment:
```suggestion
        pwSignupDiv1.lastElementChild.remove();
      const noPwSignupSpan1 = document.createElement('span');
```
span 태그만 대문자로 구분이 안되어있는 것 같아요! 👀

Reply:
헉 이 부분도 바꾸겠습니다!